owncloud/owncloud.py

```diff
@@ -804,6 +1022,105 @@ def delete_attribute(self, app, key):
             return True
         raise ResponseError(res)
 
+
+    def get_apps(self):
+    	""" List all enabled apps through the provisioning api.
+	    If this fails, try to run enable_app_ajax('provisioning_api') first.
+	    Returns a dict of apps, with values True/False, representing the enabled state.
+	"""
+	ena_apps = {}
```

Comment:
Add one more space before the `"""` block and also before `ena_apps` to align with what's below